environment.yml

```diff
@@ -0,0 +1,11 @@
+name: cm4ai-hpa
```

Comment:
Not entirely sure on this default name. What does cm4ai mean?

Reply:
CM4AI is Cell Maps for AI - the name of our DGP within Bridge2AI

Reply:
@cwinsnes Sorry that we forgot to inform you! The CM4AI project is a new large collaborative project that the lundberg lab runs with several other collaborators across the U.S. (e.g., also the Ideker lab). The purpose is to build more and better MuSIC maps. We used the HPA densenet in a hackathon that was part of the CM4AI project and @wadeschulz has been incorporating the densenet into an Azure platform for the hackathon. To do that he had to do a few changes and then I encouraged Wade to send these also as a pull request since I thought these could be helpful for other users in the future. Thank you that you replied @cwinsnes and discuss it!